#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_tload.1.dpatch by  <csmall@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Reword tload.1 man page for the alarm

@DPATCH@
diff -urNad procps-3.2.7~/tload.1 procps-3.2.7/tload.1
--- procps-3.2.7~/tload.1	2002-10-07 07:26:12.000000000 +1000
+++ procps-3.2.7/tload.1	2006-06-26 12:44:39.000000000 +1000
@@ -39,7 +39,7 @@
 .BI "\-d" " delay"
 option sets the time argument for an
 .BR alarm (2);
-if -d 0 is specified, the alarm is set to 0, which will never send the
+if \-d 0 is specified, the alarm is set to 0, which will never send the
 .B SIGALRM
 and update the display.
 
